test/Python/src/ListLiterals.lf

```diff
@@ -0,0 +1,26 @@
+// This example illustrates state variables and parameters on the wiki.
+// For this test, success is just compiling and running.
+target Python;
+
+
+reactor Sub(seq([1,2])) {
+
+}
+
+main reactor {
+    state l12([1,2]);
+    state l12_trailing([1,2,]);
+    state empty([]);
+    // doesn't parse state empty2([,]);
+
+    sub  = new Sub(seq = [1, 2]);
+    sub2 = new Sub(seq = (1, 2));
+    sub3 = new Sub(seq = ([1, 2]));
```

Comment:
This looks weird to me. What is the difference between this and line 16?

Reply:
Is `sub3` a tuple of size one that has a list in it?

Reply:
There is no difference. Parameter assignment syntax just allows that currently. You can also write `= 1` or `= (1)`.